modules/ROOT/pages/dataweave-type-reuse-xml-schema.adoc

```diff
@@ -0,0 +1,430 @@
+= Reusing an XML Schema
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can import XML schema files (`.xsd`) inside your DataWeave script as modules by using the `xmlschema!` module loader. This loader enables you to use types that are declared on your schema inside DataWeave directly.
+DataWeave loads your XSD file and translates declarations inside your file into DataWeave type directives that can be accessed in the same way as types from any other DataWeave module. Use them to build new types, type check your variables, do pattern matching, or declare new functions that use types. There are no restrictions on how to use these types.
+
+The following example shows how to look for an XSD file in the path `org/weave/myfolder/User.xsd` inside your DataWeave script:
+
+----
+import * from xmlschema!org::weave::myfolder::User
+----
+
+== Import Syntax
+
+To import the XML schema types, as shown above, use the following syntax, where:
+
+* `typeToImport`: You can use `\*` to import all types defined in the schema, or to import a single type from the schema, for example, `Root`. You can also import XML schema types with a different name, for example, `Root as Person`. This way  you can reference the type with that name in the script.
```

Comment:
```suggestion
* `typeToImport`: You can use `*` to import all types defined in the schema, or to import a single type from the schema, for example, `Root`. You can also import XML schema types with a different name, for example, `Root as Person`. This way  you can reference the type with that name in the script.
```
The \ is rendered when viewing the file in GitHub. Not sure if this will happen also in the docs page.

Reply:
Modiefied.